src/index.ts

```diff
@@ -87,6 +87,7 @@ cli.option('-w, --watch [...files]', 'watch files', { default: false })
     .option('--format [format]', 'choose one from "cjs" or "esm"')
     .option('--external [...external]', 'external')
     .option('--no-auto-external', 'if not found module, auto as external', { default: true })
+    .option('-m, --sourcemap', 'out sourcemap or not', { default: false })
```

Comment:
1. sourcemap is not commonly used, so there is no need to set a short name
2. default value shoud is undefined, it should default use farm's defaults